library/src/webapp/js/lang-datepicker/lang-datepicker.js

```diff
@@ -1436,6 +1436,14 @@ if(c&&c._defaults.timeOnly&&b.input.val()!==b.lastVal)try{$.datepicker._updateDa
  */
 !function(a){function f(a,b){if(!(a.originalEvent.touches.length>1)){a.preventDefault();var c=a.originalEvent.changedTouches[0],d=document.createEvent("MouseEvents");d.initMouseEvent(b,!0,!0,window,1,c.screenX,c.screenY,c.clientX,c.clientY,!1,!1,!1,!1,0,null),a.target.dispatchEvent(d)}}if(a.support.touch="ontouchend"in document,a.support.touch){var e,b=a.ui.mouse.prototype,c=b._mouseInit,d=b._mouseDestroy;b._touchStart=function(a){var b=this;!e&&b._mouseCapture(a.originalEvent.changedTouches[0])&&(e=!0,b._touchMoved=!1,f(a,"mouseover"),f(a,"mousemove"),f(a,"mousedown"))},b._touchMove=function(a){e&&(this._touchMoved=!0,f(a,"mousemove"))},b._touchEnd=function(a){e&&(f(a,"mouseup"),f(a,"mouseout"),this._touchMoved||f(a,"click"),e=!1)},b._mouseInit=function(){var b=this;b.element.bind({touchstart:a.proxy(b,"_touchStart"),touchmove:a.proxy(b,"_touchMove"),touchend:a.proxy(b,"_touchEnd")}),c.call(b)},b._mouseDestroy=function(){var b=this;b.element.unbind({touchstart:a.proxy(b,"_touchStart"),touchmove:a.proxy(b,"_touchMove"),touchend:a.proxy(b,"_touchEnd")}),d.call(b)}}}(jQuery);
 
+$.datepicker._updateDatepicker_original = $.datepicker._updateDatepicker;
+$.datepicker._updateDatepicker = function(inst) {
+	$.datepicker._updateDatepicker_original(inst);
+	var afterShow = this._get(inst, 'afterShow');
```

Comment:
Use const here. When we start linting these files var will fail the lint.

Reply:
It allows keyboard accessibility, not forcing the calendar to popup when typing in the text field, and set proper focus on the first input control when it pops up with the button.

Reply:
Yes, that is the aim of the PR. But I think our question on the core call was about these specific lines. What is line 1439 doing where the original is copied over?

Reply:
Is there a simpler or more clear way to do this?